express/src/services/lambda-facade/LambdaFacade.ts

```diff
@@ -107,6 +107,10 @@ export default class LambdaFacade implements LambdaFacadeInterface {
         await this.post("/send-public-beta-request-notification", JSON.stringify(body));
     }
 
+    async sendTxMALog(message: string) {
+        await this.post("/txma-logging", JSON.stringify(message));
+    }
+
```

Comment:
You Stringify the Message in the call to sendTxMALog so do you need to Stringify it again in Method?